Floofbot/Modules/Utilities.cs

```diff
@@ -16,7 +16,8 @@ public async Task Ping()
             sw.Stop();
             await msg.DeleteAsync();
 
-            EmbedBuilder builder = new EmbedBuilder() {
+            EmbedBuilder builder = new EmbedBuilder()
```

Comment:
This file seems to only have formatting changes. Maybe include it in a separate PR or a separate commit so that it's easier to undo independently.
I'd also prefer if the newline was kept at the end of the file

Reply:
reverted, was autoformat by visual studio